src/EnergyPlus/HeatBalFiniteDiffManager.hh

```diff
@@ -70,6 +70,8 @@ namespace HeatBalFiniteDiffManager {
     Real64 constexpr RhovInitValue(0.0115); // Initialization value for Rhov
     Real64 constexpr EnthInitValue(100.0);  // Initialization value for Enthalpy
     constexpr Real64 smalldiff(1.e-8);      // Used in places where "equality" tests should not be used.
+    constexpr Real64 MinTempLimit = -100.0; // lower limit check, degree C
+    constexpr Real64 MaxTempLimit = 100.0;  // upper limit check, degree C
```

Comment:
Nice.  And even further, if these constexpr are only ever used in HeatBalFiniteDiffManager.cc, they can be moved to the top of HeatBalFiniteDiffManager.cc so that they aren't exposed on the header for no reason.  No need to make that change now, but could for later.